WDAC-Policy-Wizard/app/src/Policy.cs

```diff
@@ -98,6 +108,7 @@ public WDAC_Policy()
             this._PolicyTemplate = NewPolicyTemplate.None;
             this._PolicyType = PolicyType.None;
```

Comment:
These should be default-initialized to 0, no?

Reply:
aka None

Reply:
yeah, you're right. Good catch!